common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -1991,6 +1991,10 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "Default file format for CREATE TABLE statement applied to managed tables only. External tables will be \n" +
         "created with format specified by hive.default.fileformat. Leaving this null will result in using hive.default.fileformat \n" +
         "for all tables."),
+    HIVEDEFAULTSTORAGEHANDLER("hive.default.storage.handler.class", "",
```

Comment:
Could be please use underscores to separate names? I understand that the ones before and after are not using them, but currently the underscore is the "standard"

Reply:
sure